docs/requirements.txt

```diff
@@ -7,6 +7,7 @@ geospacepy
 numpy
 OMMBV>=1.0
 pandas
+pyproject_parser
```

Comment:
Is this for readthedocs to parse the toml?

Reply:
yes

Reply:
yes.  we should look into other routes